app/assets/v2/js/pages/bounty_details.js

```diff
@@ -703,6 +727,31 @@ var show_extend_deadline_modal = function() {
   });
 };
 
+let reservedForUsername = '';
+let reservedForCreationDate = '';
+
+let set_reserved_for_link = function(bounty) {
+  const reservedFor = bounty['reserved_for_user_handle'];
+
+  if (reservedFor) {
+    reservedForUsername = reservedFor;
+    reservedForCreationDate = bounty['created_on'];
+    const reservedForHoursLeft = 72 - Math.abs(new Date() - new Date(reservedForCreationDate)) / 3600000;
+
+    // check if 24 hours have passed before setting the issue as reserved
+    if (Math.round(reservedForHoursLeft) > 0) {
+      const reservedForHtmlLink = '<a href="/profile/' + reservedFor + '">' + reservedFor + '</a>';
+      const reservedForAvatar = '<img class="rounded-circle" src="/dynamic/avatar/' + reservedFor + '" width="25" height="25"/>';
+
+      $('#bounty_reserved_for').html(reservedForHtmlLink + reservedForAvatar);
+      return;
+    }
+  }
+
```

Comment:
@mbeacom / @pinkiebell should we trigger an update to unset the reserved user if it doesn't get taken up by them ?

Reply:
I'm not biased towards the one or the other way as it doesn't change the outcome of the frontend logic. Though, it might be interesting for statistics to leave the property as is.